openforcefield/tests/test_molecule.py

```diff
@@ -688,6 +688,57 @@ def test_equality(self):
     # Test Molecule methods.
     # ----------------------
 
+    def test_find_rotatable_bonds(self):
+        """Test finding rotatable bonds while ignoring some groups"""
```

Comment:
Could you move this to `test_toolkits`, and run once with OETK and another time with RDKit? I know the different setups are _technically_ covered in our Travis testing matrix, but weird things can happen when both toolkits are installed that hide sneaky problems.

Reply:
No problem its best to be safe here, I have moved the tests.